src/main/java/com/aws/greengrass/FleetProvisioningByClaimPlugin.java

```diff
@@ -249,4 +271,19 @@ private void writeCertificateAndKeyToPath(CreateKeysAndCertificateResponse respo
             throw new RuntimeException(e);
         }
     }
+
+    private void writeCertificateToPath(CreateCertificateFromCsrResponse response, String rootPath) {
+        try {
+            Path certPath = Paths.get(rootPath, DEVICE_CERTIFICATE_PATH_RELATIVE_TO_ROOT);
+            if (Files.notExists(certPath)) {
+                Files.createFile(certPath);
+            }
+            Files.write(certPath, response.certificatePem.getBytes(StandardCharsets.UTF_8));
+            Platform.getInstance().setPermissions(FileSystemPermission.builder().ownerRead(true).build(), certPath);
+
```

Comment:
use `Permissions.OWNER_RWX_ONLY`

Reply:
I got "OWNER_RWX_ONLY is not public in com.aws.greengrass.util.Permissions;" compile error. 

I used .ownerRead(true).ownerWrite(true).ownerExecute(true) instead. Once OWNER_RWX_ONLY is public, it can be changed accordingly.